backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/add.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+from nodes.utils.utils import get_h_w_c
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:add",
+    description="Add values to an image.",
+    name="Add",
+    icon="ImBrightnessContrast",
+    inputs=[
+        ImageInput(),
+        SliderInput(
+            "Add",
+            minimum=-100,
+            maximum=100,
+            default=0,
+            precision=1,
+            controls_step=1,
+        ),
+    ],
+    outputs=[ImageOutput(image_type="Input0")],
+)
+def add_node(
+    img: np.ndarray, add: float
+) -> np.ndarray:
+    """Add values to an image"""
+
```

Comment:
We stopped documenting nodes like since the same information can be obtained from the node's description. I know that a few of the old nodes still do this, so sorry for misleading you.

```suggestion
```

Reply:
All good, will remove.